src/security/negotiation.cpp

```diff
@@ -31,6 +31,7 @@ DSN_DEFINE_bool(security,
                 enable_zookeeper_kerberos,
                 false,
                 "whether to enable kerberos for zookeeper client");
+DSN_TAG_VARIABLE(enable_zookeeper_kerberos, FT_MUTABLE);
```

Comment:
Why change it to FT_MUTABLE, isn't it immutable after the zk session has been established?